service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java

```diff
@@ -1052,12 +1052,12 @@ private boolean dontSendLinkToFondyIf(int sumToPay, Certificate certificate) {
 
     private void checkAmountOfBagsIfCourierLimitByAmountOfBag(TariffsInfo courierLocation, Integer countOfBigBag) {
         if (CourierLimit.LIMIT_BY_AMOUNT_OF_BAG.equals(courierLocation.getCourierLimit())
-            && courierLocation.getMinAmountOfBigBags() > countOfBigBag) {
+            && courierLocation.getMin() > countOfBigBag) {
```

Comment:
I think if `countOfBigBag` is equal to `courierLocation.getMin()` it should be a valid case.
For example, I as a courier set `minBags = 5`. It means that I will not proceed with an order if there are not at least 5 bags.
You as a client have 5 bags and you want to create an order. But in this case, you will see an error that `NOT_ENOUGH_BIG_BAGS_EXCEPTION`